Quadriga/src/main/java/edu/asu/spring/quadriga/domain/workspace/ITextFile.java

```diff
@@ -29,15 +29,16 @@
     public abstract String getFileContent();
 
     public abstract void setFileContent(String fileContent);
-    
+
     public abstract String getTextFileURI();
 
     /**
+     * This method sets the uri of the related TextFile based on the passed to
```

Comment:
"based on the passed" what? Also, what does "related TextFile" mean? Important here is that this method uses the id of the file as part of the URI.